journal/wagtail_hooks.py

```diff
@@ -118,6 +126,14 @@ class SciELOJournalAdmin(ModelAdmin):
         "journal__title",
         "issn_scielo",
     )
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        user_groups = request.user.groups.values_list('name', flat=True)
+        if 'Equipe de Coordenação da Coleção' in user_groups:
```

Comment:
@samuelveigarangel Adote uma constante no início do módulo de modo que se houver alguma mudança mudamos o valor da constante